spec/models/immunisation_import_spec.rb

```diff
@@ -21,7 +21,7 @@
 require "rails_helper"
 
 describe ImmunisationImport do
-  subject { described_class.create!(csv:, user:) }
+  subject(:immunisation_import) { described_class.create!(csv:, user:) }
```

Comment:
Don't think I've seen this before, cool

Reply:
Yeah I like it to make them more explicit while allowing the use the short-hand syntax `it { should }`.